runtime/queries/rmarkdown/highlights.scm

```diff
@@ -0,0 +1 @@
+; inherits: markdown
```

Comment:
Should we keep the EOF?

Reply:
Added in d2f49264920c71049a5a43d780b902f2f398df34. I copy-pasted these files from the recent ron pr (#1925), so those files are missing EOFs too if we want to be consistent about this.